packages/strapi-plugin-i18n/constants/__tests__/index.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { getInitLocale } = require('../');
+
+describe('I18N default locale', () => {
+  describe('getInitLocale', () => {
+    test('The init locale is english by default', () => {
+      expect(getInitLocale()).toEqual({
+        code: 'en',
+        name: 'English (en)',
+      });
+    });
+
+    test('The init locale can be configured by an env var', () => {
+      process.env.STRAPI_PLUGIN_I18N_INIT_LOCALE_CODE = 'fr';
+      expect(getInitLocale()).toMatchObject({
+        code: 'fr',
+      });
```

Comment:
Can you put the name too please ? It is important to have it because it is used to create the locale